activestorage/test/models/variant_test.rb

```diff
@@ -192,7 +192,7 @@ def process_variants_with(processor)
       previous_processor, ActiveStorage.variant_processor = ActiveStorage.variant_processor, processor
       yield
     rescue LoadError
-      skip "Variant processor #{processor.inspect} is not installed"
+      ENV["CI"] ? raise : skip("Variant processor #{processor.inspect} is not installed")
```

Comment:
I'm probably missing it - what's the reasoning behind this change?

Reply:
When I originally opened this PR all my vips tests were failing in the CI, even though they passed on the dev box. I got the help of @matthewd and we discovered that the CI environment did not have VIPS installed, and therefore the tests for vips were not running. He asked me to ensure that all vips tests raised errors in the CI, but were allowed to be skipped in contributor's machines to make development easier.

Reply:
If the tests are passing now, I assume that means vips has been installed in CI?

Reply:
Yes. https://github.com/rails/buildkite-config/commit/2c9273742d285784020ac8ee8ffd099c9abb6615

Reply:
Just re-approved the CI to run, let us know if theres any other blockers! :bow: